### PR TITLE
Update HAXM Documentation

### PR DESCRIPTION
**Pull request by karthikthecool**
Signed-off-by: KARTHIK VENKATESH BHAT/SRI-Bangalore-Tizen/./삼성전자 <kv.bhat@corp.samsungelectronics.net>

### CODE REVIEW

File: docs/application/tizen-studio/setup/hardware-accelerated-execution-manager.md [hunk 1]
```diff
@@ -15,39 +15,30 @@ Hardware requirements:
 <a name="on_Windows"></a>
```
Comment (varunishg) [inline docs/application/tizen-studio/setup/hardware-accelerated-execution-manager.md]: In the previous paragraph, please make the following edits. 
replace Speed up by speed-up
and speeds up by speed-ups

File: docs/application/tizen-studio/setup/hardware-accelerated-execution-manager.md [hunk 2]
```diff
@@ -15,39 +15,30 @@ Hardware requirements:
...
 ## Installing Intel&reg; HAXM on Windows&reg;
 
-Supported Windows&reg; versions: 8.1/8/7 (32/64-bit), Vista (32/64-bit), XP SP2 or higher (32-bit only)
+Supported Windows&reg; versions: 10/8.1/8/7 (32/64-bit), Vista (32/64-bit), XP SP2 or higher (32-bit only)
 
-**HAXM is installed automatically as part of the Tizen Studio**. If you want to install the HAXM individually, visit [http://download.tizen.org/sdk/haxm/6.0.4/win/](http://download.tizen.org/sdk/haxm/6.0.4/win/) and download the Windows installer package.
```
Comment (varunishg) [inline docs/application/tizen-studio/setup/hardware-accelerated-execution-manager.md]: **Rephrase** HAXM is installed automatically **to** HAXM is automatically installed 

**Rephrase** If you want to install the HAXM individually **to** If you want to install the HAXM separately

File: docs/application/tizen-studio/setup/hardware-accelerated-execution-manager.md [hunk 3]
```diff
@@ -15,39 +15,30 @@ Hardware requirements:
...
+**HAXM is installed automatically as part of the Tizen Studio**. If you want to install the HAXM individually, visit [http://download.tizen.org/sdk/haxm/7.0.0/win/](http://download.tizen.org/sdk/haxm/7.0.0/win/) and download the Windows installer package.
 
 **Table: Windows installer package**
 
-| Windows installer                        | File description | File size | MD5 checksum                     |
+| Windows installer                        | File description | File size | SHA256SUM                     |
 |----------------------------------------|----------------|---------|--------------------------------|
-| [IntelHAXM_6.0.4.exe](http://download.tizen.org/sdk/haxm/6.0.4/win/IntelHAXM_6.0.4.exe) | System driver    | 2.66 MB   | bc05f881d9282ed0914b3e0bf3c96e89 |
+| [IntelHAXM_7.0.0.exe](http://download.tizen.org/sdk/haxm/7.0.0/win/IntelHAXM_7.0.0.exe) | System driver    | 2.89 MB   | 1587f7ae42e11c7b22dc674fd776ce252b6a19152e7ca416a748e2bdbf2b36c5 |
```
Comment (varunishg) [inline docs/application/tizen-studio/setup/hardware-accelerated-execution-manager.md]: In my opinion there is no point in giving the same link twice in the same section. 
Please remove the URL from the table and keep the filename in BOLD. 
You can give the full path to the URL in the previous paragraph.

File: docs/application/tizen-studio/setup/hardware-accelerated-execution-manager.md [hunk 4]
```diff
@@ -15,39 +15,30 @@ Hardware requirements:
...
-
-   ![Adjust RAM](./media/hardware_ram.png)
-
-   The installer also functions as a configuration tool for Intel&reg; HAXM. To change the memory settings later, run the installer again.
-
-   > **Note**  
-   > If you change the Intel&reg; HAXM memory settings for the emulator, you must restart the Intel&reg; HAXM and the emulator. Emulators currently running continue to use the previous memory setting.
-
-4. Confirm your Intel&reg; HAXM memory allocation settings and click **Install**.  
+3. **Install** will trigger the installation
```
Comment (varunishg) [inline docs/application/tizen-studio/setup/hardware-accelerated-execution-manager.md]: Rephrase: **Install** will trigger the installation to The following screen shows the installation progress. 

Also remove the numbering from this step. Let it be a continuing part of #2.

File: docs/application/tizen-studio/setup/hardware-accelerated-execution-manager.md [hunk 5]
```diff
@@ -15,39 +15,30 @@ Hardware requirements:
...
-
-   > **Note**  
-   > If you change the Intel&reg; HAXM memory settings for the emulator, you must restart the Intel&reg; HAXM and the emulator. Emulators currently running continue to use the previous memory setting.
-
-4. Confirm your Intel&reg; HAXM memory allocation settings and click **Install**.  
+3. **Install** will trigger the installation  
 ![Confirm installation](./media/hardware_instal.png)
 
-5. When the Intel&reg; HAXM installation is finished, click **Finish** to exit the installer.  
+4. When the Intel&reg; HAXM installation is finished, click **Finish** to exit the installer.
```
Comment (varunishg) [inline docs/application/tizen-studio/setup/hardware-accelerated-execution-manager.md]: Rephrase as following: 
Click **Finish** to exit the installer, once the Intel&reg; HAXM installation is finished.

Reason being - We always follow the rule of Action first, reason/outcome later.

File: docs/application/tizen-studio/setup/hardware-accelerated-execution-manager.md [hunk 6]
```diff
@@ -15,39 +15,30 @@ Hardware requirements:
...
-
-4. Confirm your Intel&reg; HAXM memory allocation settings and click **Install**.  
+3. **Install** will trigger the installation  
 ![Confirm installation](./media/hardware_instal.png)
 
-5. When the Intel&reg; HAXM installation is finished, click **Finish** to exit the installer.  
+4. When the Intel&reg; HAXM installation is finished, click **Finish** to exit the installer.  
 ![Finish](./media/hardware_finish.png)
 
 The Intel&reg; HAXM is installed and ready to use.
```
Comment (varunishg) [inline docs/application/tizen-studio/setup/hardware-accelerated-execution-manager.md]: Rephrase the following sentence: 
Execute the following command, to verify that Intel&reg; HAXM is running.

Comment (varunishg) [reply docs/application/tizen-studio/setup/hardware-accelerated-execution-manager.md]: Line #77  To uninstall the Intel&reg; HAXM, run the installer again or use the Control Panel. 
**Rephrase to**  Run installer or use control panel, to uninstall the Intel&reg; HAXM.
Line #86, 87, 88 and 89 > rephrase Make Sure to Ensure.

File: docs/application/tizen-studio/setup/hardware-accelerated-execution-manager.md [hunk 7]
```diff
@@ -97,15 +88,15 @@ If you meet an installation failure because of hardware requirements, check the
 <a name="on_MacOS"></a>
 ## Installing Intel&reg; HAXM on macOS
 
-Supported macOS versions: 10.10 Yosemite (64-bit), 10.9 Mavericks (64-bit), 10.8 Mountain Lion (64-bit)
+Supported macOS versions: 10.12 Sierra (64-bit), 10.11 OS X El Capitan(64-bit), 10.10 Yosemite (64-bit), 10.9 Mavericks (64-bit), 10.8 Mountain Lion (64-bit)
 
-**HAXM is installed automatically as part of the Tizen Studio.** If you want to install the HAXM individually, visit [http://download.tizen.org/sdk/haxm/6.0.3/mac/](http://download.tizen.org/sdk/haxm/6.0.3/mac/) and download the macOS installer package.
+**HAXM is installed automatically as part of the Tizen Studio.** If you want to install the HAXM individually, visit [http://download.tizen.org/sdk/haxm/7.0.0/mac/](http://download.tizen.org/sdk/haxm/7.0.0/mac/) and download the macOS installer package.
```
Comment (varunishg) [inline docs/application/tizen-studio/setup/hardware-accelerated-execution-manager.md]: **Rephrase** HAXM is installed automatically **to** HAXM is automatically installed

**Rephrase** If you want to install the HAXM individually **to** If you want to install the HAXM separately

File: docs/application/tizen-studio/setup/hardware-accelerated-execution-manager.md [hunk 8]
```diff
@@ -97,15 +88,15 @@ If you meet an installation failure because of hardware requirements, check the
...
 
 **Table: Windows installer package**
 
-| macOS installer                          | File description | File size | MD5 checksum                     |
+| macOS installer                          | File description | File size | SHA256SUM                     |
 |----------------------------------------|----------------|---------|--------------------------------|
-| [IntelHAXM_6.0.3.dmg](http://download.tizen.org/sdk/haxm/6.0.3/mac/IntelHAXM_6.0.3.dmg) | System driver    | 213 KB    | cbd7e3cbdb8b4a84c91fc9094b2cd134 |
+| [IntelHAXM_7.0.0.dmg](http://download.tizen.org/sdk/haxm/7.0.0/mac/IntelHAXM_7.0.0.dmg) | System driver    | 218 KB    | fe7e4cd86b7a2b85591397fd6bf2ef43e7b088bfc72badab0c4d3532a65c2a2f |
```
Comment (varunishg) [inline docs/application/tizen-studio/setup/hardware-accelerated-execution-manager.md]: In my opinion there is no point in giving the same link twice in the same section.
Please remove the URL from the table and keep the filename in BOLD.
You can give the full path to the URL in the previous paragraph.

**Comment by varunishg** [comment]
All review comments have been incorporated.

File: docs/application/tizen-studio/setup/hardware-accelerated-execution-manager.md [hunk 9]
```diff
@@ -9,50 +9,45 @@ Hardware requirements:
...
-5. When the Intel&reg; HAXM installation is finished, click **Finish** to exit the installer.  
-![Finish](./media/hardware_finish.png)
+3. Click **Finish** to exit the installer, once the Intel&reg; HAXM installation is finished.
+
+	![Finish](./media/hardware_finish.png)
 
 The Intel&reg; HAXM is installed and ready to use.
 
-To verify that Intel&reg; HAXM is running, open the Command Prompt window and execute the following command:
+Execute the following command, to verify if Intel® HAXM is running:
```
Comment (mijin85cho) [inline docs/application/tizen-studio/setup/hardware-accelerated-execution-manager.md]: In the md file, please use use `&reg;` to show ®.

File: docs/application/tizen-studio/setup/hardware-accelerated-execution-manager.md [hunk 10]
```diff
@@ -74,7 +69,7 @@ To start or stop Intel&reg; HAXM, open the Command Prompt window with administra
 > **Warning**  
 > Close all instances of the Tizen emulator before removing the Intel&reg; HAXM.
 
-To uninstall the Intel&reg; HAXM, run the installer again or use the Control Panel.
+Run installer or use control panel, to uninstall the Intel® HAXM.
```
Comment (mijin85cho) [inline docs/application/tizen-studio/setup/hardware-accelerated-execution-manager.md]: In the md file, please use use `&reg;` to show ®.

File: docs/application/tizen-studio/setup/hardware-accelerated-execution-manager.md [hunk 11]
```diff
@@ -9,50 +9,45 @@ Hardware requirements:
 - Intel processor with support for VT-x, EM64T, and Execute Disable Bit
 - At least 1 GB of available memory (2 GB or more is preferred)
 
-> **Important**  
+> **Important**
```
Comment (mijin85cho) [inline docs/application/tizen-studio/setup/hardware-accelerated-execution-manager.md]: In the md file, to make a line break. you should keep two spaces after `**Important**`. Please add one deleted space back again.

**Comment by mijin85cho** [comment]
I checked the latest update, and found few more points to fix. As varunishg checked other things, I'll merge this PR if they are fixed.

**Comment by mijin80lee** [comment]
TD Sync Completed.

**Pull request merged**